### DISCUSSION TITLE
Basic Question: How to use the Camply Library in Jupyter Notebook

### DISCUSSION THREAD

**Discussion by purnanandsamant** [juftin/camply#212] [General]
Once I import Camply, how do I use the code. I am new to Python and I am not able to run the next steps.

Code:

Import camply

**Comment by juftin** [comment]
Camply is first and foremost an application meant to be used from the command line. Its usage as Python library is less documented, but you can certainly use it that way too.

- [Python Usage Documentation](https://juftin.com/camply/python/)

- [Python API Documentation](https://juftin.com/camply/reference/)

If you're having trouble importing camply, it's likely not installed into your Python environment (kernel) running the Jupyter notebook.

You can run the install command from the command line or from inside a Jupyter Cell (inside Jupyter is the one with the exclamation point):

```shell
pip install camply
```

```shell
!pip install camply
```

Python environments are difficult when you first start out. Without knowing much about your local development environment it's hard to know exactly what's going wrong. Here's a separate article I found about installing dependencies into Jupyter: https://jakevdp.github.io/blog/2017/12/05/installing-python-packages-from-jupyter/. 

Hope this helps. Good luck!
